src/main/webapp/courseLoader.js

```diff
@@ -238,27 +237,31 @@
   async function getRatingPropertiesToStore() {
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
+    let wantsToTranslate;
+
+    if (document.getElementById("true").checked) {
```

Comment:
why don't you just have one box that the user can check if they want to translate? that way you don't have to deal with a case where someone checks both or no boxes

Reply:
Decided on a dropdown menu instead so now I just have to call document.getElementById("translate").